index.js

```diff
@@ -54,4 +54,41 @@ function createList() {
     const element = document.querySelector('.main')
     element.appendChild(newList)
 
+    // ********************* adding a title for movie in the list *************************** //
+
+    // adding an event listener to this created inputBtn that is inside the beginning binge list box
+inputBtn.addEventListener('click', (e) => {
+  
+  //get the movie title from the input.
+
+  // once the user enters a new movie name, then make a new
+  if(inputTitle.value !== "" && inputTitle.value !== new RegExp("/^\s+$/")) {
```

Comment:
Awesome that you've added this! 🕺 Makes our App seem so PRO!

Reply:
Nice use of regex!

Reply:
Thank you ✨